uui-core/src/data/processing/views/__tests__/Tree.load.test.ts

```diff
@@ -61,14 +61,14 @@ describe('Tree - load', () => {
     });
 
     function expectTreeToLookLike(
-        actual: Tree<TestItem, number>,
+        actual: ITree<TestItem, number>,
```

Comment:
why we change interface here and then lose type checking? it will be not safe, if we change this methods in future.

Reply:
All the methods of the Tree return ITree as the result. If the type of the `actual` variable will be `Tree`, we'll receive a type error while passing the ITree (produced by any method of Tree class) as an arg. We have such a situation in develop, which is wrong.
![Screenshot 2023-07-18 at 09 50 18](https://github.com/epam/UUI/assets/22456368/0ea02438-b374-43f7-b279-ded6142a18be)
![Screenshot 2023-07-18 at 09 50 41](https://github.com/epam/UUI/assets/22456368/02efb040-37b6-49b6-86e4-a98874181d42)
![Screenshot 2023-07-18 at 09 50 49](https://github.com/epam/UUI/assets/22456368/37f6a86b-f822-443d-904c-43341a8ccb10)